tests/python_tests/ov_genai_test_utils.py

```diff
@@ -173,6 +173,19 @@ def load_genai_pipe_with_configs(configs: List[Tuple], temp_path):
     # remove existing jsons from previous tests
     for json_file in temp_path.glob("*.json"):
         json_file.unlink()
+    # rt_info has the highest priority. Delete it to respect configs.
+    core = openvino.Core()
+    core.set_property({'ENABLE_MMAP': False})
+    for model_path in temp_path / "openvino_tokenizer.xml", temp_path / "openvino_detokenizer.xml":
+        tokenizer = core.read_model(model_path)
+        rt_info = tokenizer.get_rt_info()
+        for config, _ in configs:
+            for key in config.keys():
+                try:
+                    del rt_info[key]
+                except KeyError:
+                    pass
+        openvino.save_model(tokenizer, model_path)
```

Comment:
looks like it can be extracted to a separate function as it's used twice already